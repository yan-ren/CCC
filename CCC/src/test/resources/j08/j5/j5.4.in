5
8 7 8 4
7 7 7 8
8 8 8 8
7 8 7 7
8 0 8 8
