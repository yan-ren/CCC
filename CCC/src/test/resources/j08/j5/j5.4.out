Roland
Patrick
Patrick
Roland
Roland
